src/services/storage-service.ts

```diff
@@ -0,0 +1,63 @@
+import { Note } from '../types';
+
+export class StorageService {
+    private readonly STORAGE_KEY_PREFIX = 'note_';
+
+    private getKey(id: string): string {
+        return this.STORAGE_KEY_PREFIX + id;
+    }
+
+    getAllNotes(): Note[] {
```

Comment:
I would add another method or a parameter to get removed notes only, but not all of them together.